specification/out/index.html

```diff
@@ -228,11 +228,18 @@ <h1><a name="Patients">Patients</a></h1>
   <div class="method"><a name="acceptance"/>
     <div class="method-path">
     <a class="up" href="#__Methods">Up</a>
-    <pre class="post"><code class="huge"><span class="http-method">post</span> /Patient/{id}</code></pre></div>
+    <pre class="post"><code class="huge"><span class="http-method">post</span> /Patient/{id}/$nhais.acceptance</code></pre></div>
```

Comment:
should we remove {id} ?
'Change API to POST /fhir/Patient/$nhais.acceptance'

Reply:
good catch, updated now